tools/downloader/converter.py

```diff
@@ -56,19 +56,25 @@ def prefixed_subprocess(prefix, args):
             sys.stdout.write(prefix + ': ' + line)
         return p.wait() == 0
 
+
 def convert_to_onnx(model, output_dir, args, stdout_prefix):
-    pytorch_converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
-    prefixed_printf(stdout_prefix, '========= {}Converting {} to ONNX',
-        '(DRY RUN) ' if args.dry_run else '', model.name)
-
-    pytorch_to_onnx_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
-                                                            dl_dir=args.download_dir / model.subdirectory)
-                            for arg in model.pytorch_to_onnx_args]
-    cmd = [str(args.python), str(pytorch_converter), *pytorch_to_onnx_args]
+    prefixed_printf(stdout_prefix, '========= {}Converting {} to ONNX from {}',
+                    '(DRY RUN) ' if args.dry_run else '', model.name, model.framework)
+
+    conversion_to_onnx_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
+                                                               dl_dir=args.download_dir / model.subdirectory)
+                               for arg in model.conversion_to_onnx_args]
+    if model.framework=='pytorch':
```

Comment:
May be it would be better to add some variable like `common.KNOWN_FRAMEWORKS` for frameworks which support ONNX conversion and use it here.

Reply:
You would still have to pick a converter depending on the framework, so it wouldn't help much.

Reply:
you can use dictionary for selection, where key is framework and function for call converter realisation like:
```python
onnx_converters = {
'pytorch': pytorch_to_onnx,
'caffe2': caffe2_to_onnx
}
converter = onnx_converters.get(model.framework)
if converter is None:
  # handle error of unknown converter
```
}